test/test_core.py

```diff
@@ -453,59 +453,78 @@ def test_int53_convertI32PairToI53Checked(self):
       self.do_core_test('test_convertI32PairToI53Checked.cpp', interleaved_output=False)
 
   def test_i64(self):
+    # test shifts etc. on 64-bit integers as well as printf() on them. we need
+    # the math testing only for wasm2js but do not apply @only_wasm2js since we
+    # do want some testing of 64-bit printf in our libc (which is not tested in
+    # clang upstream).
```

Comment:
I believe we test printf elsewhere (in fact we might have many overlapping printf tests).  e.g. test/printf/test.c

Reply:
Good point, removed this one too.